editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -865,3 +868,41 @@ export function useClearKeyboardInteraction(editorStoreRef: {
     window.addEventListener('mousedown', clearKeyboardInteraction, { once: true, capture: true })
   }, [dispatch, editorStoreRef])
 }
+
+export function useHighlighPaddingHandlers(): {
+  onMouseEnter: () => void
```

Comment:
If you were to change this returned callback to take an array of sides then this could be reused in the `PaddingControl`. Or if not, the below `paddingControlsForHover` can just be an array, since you're immediately turning it back into an array anyway

Reply:
great idea! implemented in [41a28a9](https://github.com/concrete-utopia/utopia/pull/3265/commits/41a28a9b93bb1bf373e389c8e4ff07599eb1ecfc)